src/Fade.ts

```diff
@@ -1,45 +1,69 @@
-import Flicking, { FlickingEvent, Plugin } from "@egjs/flicking";
+import Flicking, { Plugin } from "@egjs/flicking";
 
 /**
  * You can apply fade in / out effect while panel is moving.
  * @ko 패널들을 움직이면서 fade in / out 효과를 부여할 수 있습니다.
- * @memberof eg.Flicking.plugins
+ * @memberof Flicking.Plugins
  */
 class Fade implements Plugin {
+  private _flicking: Flicking | null;
+
+  /* Options  */
+  private _selector: string;
+  private _scale: number;
+
   /**
    * @param - The selector of the element to which the fade effect is to be applied. If the selector is blank, it applies to panel element. <ko>Fade 효과를 적용할 대상의 선택자. 선택자가 공백이면 패널 엘리먼트에 적용된다.</ko>
-   * @param - Effect amplication scale <ko>효과 증폭도</ko>
+   * @param - Effect amplication scale<ko>효과 증폭도</ko>
    * @example
-   * flicking.addPlugins(new eg.Flicking.plugins.Fade("p", 1));
+   * flicking.addPlugins(new Fade("p", 1));
    */
-  constructor(private selector: string = "", private scale: number = 1) {}
-  public init(flicking: Flicking): void {
-    flicking.on("move", this.onMove);
-    this.move(flicking);
+  public constructor(selector = "", scale = 1) {
+    this._flicking = null;
+    this._selector = selector;
+    this._scale = scale;
   }
-  public update(flicking: Flicking): void {
-    this.move(flicking);
+
+  public init(flicking: Flicking): void {
+    if (this._flicking) {
+      this.destroy();
+    }
+
+    this._flicking = flicking;
+
+    flicking.on("move", this._onMove);
+    this._onMove();
```

Comment:
It seems that the code to initialize has changed from the constructor to the init method. Are there any issues that users switching from v3 to v4 feel? And is there no need to add a plugin guide for this?

Reply:
Nope, initialization was done in the `init()` before.
https://github.com/naver/egjs-flicking-plugins/blob/master/src/Fade.ts#L16

Reply:
Aha. Yes. I saw the wrong one line of code.